src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -355,12 +357,7 @@ Result AudioStreamAAudio::close() {
             // Make sure we are really stopped. Do it under mLock
             // so another thread cannot call requestStart() right before the close.
             requestStop_l(stream);
```

Comment:
It's a bit inconsistent that requestStop_l() in AAudio is in OboeGlobals::areWorkaroundsEnabled() but requestPause_l() in OpenSL ES isn't. Should we change one of them to make them consistent?

Reply:
True. I will change the openSL code.

Reply:
On second thought. This could cause a regression in some app that has disabled workarounds when using OpenSL ES.  The AAudio code is not supposed to need the pause but we found it does so we consider the pause a workaround. I will leave as-is. But I noticed I forgot to call the sleep from the Input side so I added that.